lib/Drush/Commands/core/BrowseCommands.php

```diff
@@ -30,7 +35,19 @@ public function browse($path = '', $options = ['browser' => NULL]) {
     $alias = drush_get_context('DRUSH_TARGET_SITE_ALIAS');
     if (drush_sitealias_is_remote_site($alias)) {
       $site_record = drush_sitealias_get_record($alias);
-      $return = drush_invoke_process($site_record, 'browse', [$path], drush_redispatch_get_options(), array('integrate' => TRUE));
+
+      // Experiment 1a.
+      $ret = $this->taskExec('drush browse')->run();
```

Comment:
This works but it looks ugly. Better to pass an object to taskExec()? See next experiment.